core/components/header/_actions/logout.ts

```diff
@@ -1,7 +1,11 @@
 'use server';
 
+import { redirect } from 'next/navigation';
+
 import { signOut } from '~/auth';
 
 export const logout = async () => {
-  await signOut();
+  await signOut({ redirect: false });
+
+  redirect('/login');
```

Comment:
I think that `signOut` takes `redirectTo` as an option also. can we use it instead of `redirect` method?

Reply:
I used this approach to be [consistent](https://github.com/bigcommerce/catalyst/blob/main/core/app/%5Blocale%5D/(default)/login/register-customer/_components/register-customer-form/_actions/login.ts#L17)  with Chance's approach to login